pylint/checkers/format.py

```diff
@@ -466,7 +467,7 @@ def process_tokens(self, tokens: list[tokenize.TokenInfo]) -> None:
         if line_num == last_blank_line_num and line_num > 0:
             self.add_message("trailing-newlines", line=line_num)
 
-    def _check_line_ending(self, line_ending, line_num):
+    def _check_line_ending(self, line_ending: str, line_num: int) -> None:
```

Comment:
While we're at it, would you mind renaming `token` on line L396 and L414 to `string`? It what's being used to call this function and originally threw me off. The second argument of `TokenInfo` is called a `string` and not a `token`.